### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by seanrcollings**
A lot of the documentation has drifted hopelessly out of date, this aims to fix that!

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -148,15 +148,15 @@
       {
         # Canvas
         iss: "https://canvas.instructure.com",
-        client_id: "43460000000000378",
+        client_id: secrets.canvas_developer_id,
```
Comment (jbasdf) [inline db/seeds.rb:151]: Did you add a new value to the secrets? This value is the id Canvas provides when you setup a new LTI Advantage app. Make sure this isn't the developer id associated with the OAuth key and secret.

File: template.rb [hunk 2]
```diff
@@ -151,15 +152,15 @@ def app_dir
...
+    app_name.underscore.upcase
   end
 
   gsub_file(f, "hello_world") do |_match|
     app_name.underscore
   end
 
   gsub_file(f, "helloworld") do |_match|
-    app_name.underscore
+    url_safe_name
```
Comment (jbasdf) [inline template.rb:163]: Have run the the template to generate a new project to make sure evey thing still works?

File: db/seeds.rb (hunk 1 above)
Comment (seanrcollings) [reply db/seeds.rb:151]: image
Aren't these equivalent? The README said to put the ID and Key provided by canvas into your .env file, with the ID being the same value we use when installing an LTI Advantage app. Unless I'm misunderstanding, those two should be equivalent?

Comment (jbasdf) [reply db/seeds.rb:151]: They are different. The one in this file is used as the id for the LTI Advantage application. The other is used to let a user do the OAuth dance with our application so we can get a Canvas API keyl

File: template.rb (hunk 2 above)
Comment (seanrcollings) [reply template.rb:163]: Yep, it all should be working fine. Tests all passed and I got it to run and everything

**Review by jbasdf — APPROVED** [review]

**Pull request merged**